app/assets/javascripts/discourse/app/components/ace-editor.gjs

```diff
@@ -278,6 +284,18 @@ export default class AceEditor extends Component {
           ...attributes
         >
         </div>
+        {{#if @resizable}}
+          <div
+            class="grippie"
+            {{grippieDragResize
+              ".ace_editor--resizable"
+              "bottom"
+              this.onResizeDragStart
+              this.onResizeDrag
```

Comment:
I only see this function, where are the others?

Reply:
They are not needed here, will `null` them, I missed this